### PR TITLE
Test case for Profiler

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Fixes 5) in https://github.com/cogneco/ooc-kean/issues/716
@davidhesselbom  can you review ?

### CODE REVIEW

File: test/base/ProfilerTest.ooc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+use ooc-base
+use ooc-unit
+import io/File
+
+ProfilerTest: class extends Fixture {
+	init: func {
+		super("Profiler")
+		this add("log", func {
+			outputFile := "test/test.log"
```
Comment (davidhesselbom) [inline test/base/ProfilerTest.ooc]: How about `test/profilerTest.log`?

Comment (sebastianbaginski) [inline test/base/ProfilerTest.ooc]: OK, but this file is removed in this test. Should we keep it for manual inspection ?

Comment (davidhesselbom) [inline test/base/ProfilerTest.ooc]: Also, I noticed in another test that you created strings as member variables and freed them in a `free` function. Any reason not to do that here?

Comment (davidhesselbom) [inline test/base/ProfilerTest.ooc]: > OK, but this file is removed in this test. Should we keep it for manual inspection ?

Oh, I missed that. Still, no harm in using more specific file names, is there?

Comment (davidhesselbom) [inline test/base/ProfilerTest.ooc]: And yes, we can keep it, why not.

Comment (sebastianbaginski) [inline test/base/ProfilerTest.ooc]: OK, updated

**Comment by marcusnaslund** [comment]
I can take this one, it's simple enough.

**Comment by marcusnaslund** [comment]
If you can think of a nice way to test `reset` and `resetAll` we would have a complete test. If you're busy with other stuff, this is good enough for now.

**Comment by sebastianbaginski** [comment]
If it's possible then I'd prefer to do that later

**Comment by marcusnaslund** [comment]
Fine with me.

**Pull request merged**